### PR TITLE
Add ui playground by storybook

### LINKED ISSUE #593: ui: create playground page by storybook
## Feature Request

**Is your feature request related to a problem? Please describe:**

Related PR: #591 

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by baurine**
close #593

### CODE REVIEW

**Comment by baurine** [comment]
Current progress:

image

Remain a big issue: how to set the API client instance for all stories, still working on it.

**Comment by baurine** [comment]
image

Almost done, remain some cleanup work.

**Comment by baurine** [comment]
Final effects:

企业微信20200717094713

企业微信20200717094817

Btw, ideally, we should use mocked data for storybook to avoid to depend on an extra service but here we use the real data, I think it is just fine, and maybe we can change it in the later gradually. How do you think about it? @breeswish

File: ui/lib/components/MultiSelect/index.tsx [hunk 1]
```diff
@@ -47,9 +47,7 @@ export interface IMultiSelectProps<T>
   columnTitle?: string
 }
 
-export default function MultiSelect<T extends IItem>(
-  props: IMultiSelectProps<T>
-) {
+function MultiSelect<T extends IItem>(props: IMultiSelectProps<T>) {
```
Comment (baurine) [inline ui/lib/components/MultiSelect/index.tsx:50]: Changing to this writing style to pass the storybook compilation. Ref: https://github.com/storybookjs/storybook/issues/11419#issuecomment-658969643

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,9 +26,9 @@ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh
 
 Declare the global environment variable:
 
-   > **Note:**
-   >
-   > After the installation, TiUP displays the absolute path of the corresponding `profile` file. You need to modify the following `source` command according to the path.
+> **Note:**
+>
+> After the installation, TiUP displays the absolute path of the corresponding `profile` file. You need to modify the following `source` command according to the path.
```
Comment (baurine) [inline CONTRIBUTING.md:31]: Auto formatted by the git commit hook.

**Comment by baurine** [comment]
@breeswish PTAL when you're free, thanks!

File: ui/package.json [hunk 3]
```diff
@@ -48,7 +48,9 @@
     "build": "gulp build",
     "test": "react-app-rewired test",
     "fmt": "prettier --write .",
-    "gen:browserlist": "gulp gen:browserlist"
+    "gen:browserlist": "gulp gen:browserlist",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public"
```
Comment (breeswish) [inline ui/package.json:53]: Maybe we don't need `build-storybook`

File: ui/lib/components/BaseSelect/1-antd-select.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```
Comment (breeswish) [inline ui/lib/components/BaseSelect/1-antd-select.stories.tsx:1]: We can drop the standard ant selects

File: ui/package.json (hunk 3 above)
Comment (baurine) [reply ui/package.json:53]: I think so, will remove it soon.

File: ui/lib/components/BaseSelect/1-antd-select.stories.tsx (hunk 4 above)
Comment (baurine) [reply ui/lib/components/BaseSelect/1-antd-select.stories.tsx:1]: Ok.

**Pull request merged**